### ISSUE TITLE
Travis CI automatic releases

### ISSUE THREAD

**Issue by Polpetta** [Polpetta/jkk#17]
Ok, so after #15 we have now .debs and .rpms. Although we don't have a really working version yet, we can push our binaries in the Github releases section via Travis CI (https://docs.travis-ci.com/user/deployment/releases#uploading-multiple-files)

This will enable us to have binaries automatically published once we push a new git tag! In this way if people want to try it out without messing with package repositories they can easily.

Finally, we need to check out this bot https://github.com/apps/release-drafter that draft releases for us while we merge PRs (it automatically creates changelogs and stuff), in particular we need to check if it works with Travis release deployment!

**Comment by M0Rf30** [comment]
Now we need to check the adoption of release-drafter. IMHO we could move this choice to another issue and close this

**Comment by Polpetta** [comment]
Ok boss! See #26 :wink:
